src/components/contentful/contentfulOpenLetterForm/contentfulOpenLetterForm.js

```diff
@@ -51,12 +52,7 @@ const OpenLetterForm = ({ content }) => {
         {!expandOpenLetterStart && !expandOpenLetterRead && (
           <>
             {description && (
-              <div
-                className="ContentBox__content"
-                dangerouslySetInnerHTML={{
-                  __html: description.childMarkdownRemark.html,
-                }}
-              />
+              <div className="ContentBox__content">{documentToReactComponents(description.raw)}</div>
```

Comment:
Unfortunately the description field is still using the old Contentful markdown field, so we can't really use `documentToReactComponents` - which only supports rich text field. So this will always return empty even when the description is set. 

There is a mention on migrating markdown to rich text here https://www.contentful.com/developers/docs/javascript/tutorials/rendering-contentful-rich-text-with-javascript/#migrating-from-markdown-to-rich-text, but it seems like a rather tedious work-around that coerce rich text field to support the `documentToReactComponents` function. I guess the other option (and probably more sensible IMO) is to change all markdown fields to rich text fields from Contentful model. But that would require copy-pasting all the existing markdown content to the rich text field. I created a refactor issue here https://github.com/Naisten-Linja/naisten-linja-public-website/issues/56, so it's not forgotten later.